### ISSUE TITLE
About the additional layers

### ISSUE THREAD

**Issue by zhangmo123** [lpzjerry/Pedestrian-Attribute-LGNet#1]
Hi
     Thank you for the source code, im very interesting about your work。
     Would you share the addition layers？

**Comment by lpzjerry** [comment]
Hi Mo,
Thanks for your interest. Pitifully, the Caffe that I used to build this project is not accessible now. That is the reason why I highlight the mechanism of "CalOverlap" and "CamBox" layer in README.md.

Since this project is based on an out-of-date version of Caffe, I am planning to replicate this work on a more advanced DL framework (most probably PyTorch), hopefully in this summer. Please pay attention to this repository for any updates.

You are also welcome to reproduce the CalOverlap and CamBox layer. (Tips: both layers do not require backpropagation.) The weighted loss layer can be adapted from the original Sigmoid cross entropy layer. Please feel free to contact me again if anything is unclear.

**Comment by wait1988** [comment]
Mark
